src/backend/backend.ts

```diff
@@ -30,13 +30,6 @@ window.addEventListener('message', function(event) {
 
     if (event.data.message.message.actionType ===
```

Comment:
the message.message is not a part of this PR, but this is a leftover from the days when I used to contribute to this :), we should unnest this sometime in the future.